### PR TITLE
GH-1090: Footer

### PR DESCRIPTION
**Pull request by bhryck**

### CODE REVIEW

File: ui/src/components/footer/Footer.js [hunk 1]
```diff
@@ -0,0 +1,316 @@
+import {
+  Link,
+  Container,
+  Image,
+  Text,
+  VStack,
+  Center,
```
Comment (ksuhartono97) [inline ui/src/components/footer/Footer.js:7]: can remove unused imports

File: ui/src/components/footer/Footer.js [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+} from '@chakra-ui/react';
+import './SocialIcon.css';
+import { EmailIcon } from '@chakra-ui/icons';
+import { SocialIcon } from 'react-social-icons';
+
+const Footer = () => {
+  return (
+    <Container maxW="container.xl" py="2em">
+      <Flex w="100%" direction={['column', 'row']} py="2em">
+        <VStack w={['100%', '40%']} alignItems={['center', 'flex-start']}>
```
Comment (ksuhartono97) [inline ui/src/components/footer/Footer.js:24]: space between items should appear wider according to design. similar situation for mobile

Current:
image

Design:
image


you can just add a `spacing={}` here to do this.

but to keep the logo and text close you will need to wrap it
```
<Stack flexDir="column">
  <Link href="/">
    <Image
      src={`${process.env.PUBLIC_URL}/images/ripple_blue.svg`}
      alt="Logo of HMCC"
      h="2em"
    />
  </Link>
  <Text
    color="#4A6EEB"
    fontSize="16"
    fontWeight="700"
    fontFamily="Manrope"
    wordWrap="break-word"
    textAlign={['center', 'left']}
  >
    Harvest Mission Community Church <br /> Hong Kong
  </Text>
</Stack>
```
something like that

**Review by ksuhartono97 — CHANGES_REQUESTED** [review]
Font thickness looks off but i think that's because this branch hasn't rebased from release-web-revamp. make sure to do that and i think it should be ok after that

File: ui/src/components/footer/Footer.js (hunk 2 above)
Comment (bhryck) [reply ui/src/components/footer/Footer.js:22]: added padding to make it look more similar

**Review by ksuhartono97 — APPROVED** [review]

**Pull request merged**